web/src/components/shared/bestComponent/BestComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext, useState } from 'react';
+import { PageResourceContext } from '../../../contexts';
+import { SaleItem } from '../saleItem/SaleItem';
+import './style.css';
+
+const BestComponent = (): JSX.Element => {
+    const {
+        pageResource: { products },
+    } = useContext(PageResourceContext);
+    const [limit, setLimit] = useState(6);
```

Comment:
actually limis is contance. just create a variable outside this function (line 5) and do 

const BEST_LIMIT = 6;